src/Classes/Orchestrator/Choria.php

```diff
@@ -208,6 +215,23 @@ public function removeInstance()
         return ServerUtility::executeShellCommand($this->parseCommand(self::$removeNodeCommand, false, [$this->instance->getRunnerId()]));
     }
 
+    public function dispatchReplicas(array $executionsWithNewReplicaCount)
+    {
+        $executionReplicasArray = [];
+        /** @var Execution $execution */
+        foreach ($executionsWithNewReplicaCount as $execution) {
+            $replica = $execution->getReplicas() ?? JobFactory::getDispatchConfigOfJob($this->job, $execution)->getDispatchConfig()->getReplicaCountForJob($this->job);
+            $executionReplicasArray[] = [
+                'service' => $execution->getServiceName(),
+                'scale' => $replica,
+            ];
+        }
+
+        $command = $this->parseCommand(self::$serviceScaleCommand, false, [$this->instance->getRunnerId()]);
+
+        return ServerUtility::executeShellCommand(str_replace('$serviceScaleArray', json_encode($executionReplicasArray), $command));
```

Comment:
```suggestion
        return ServerUtility::executeShellCommand(str_replace('{{serviceScaleArray}}', json_encode($executionReplicasArray), $command));
```

if you accept above suggestion, here is the counterpart ;)